lib/utils/normalizeOptions.js

```diff
@@ -9,6 +9,12 @@ function normalizeOptions(compiler, options) {
   options.contentBase =
     options.contentBase !== undefined ? options.contentBase : process.cwd();
 
+  // Setup default value
+  options.contentBasePublicPath =
+    options.contentBasePublicPath !== undefined
+      ? options.contentBasePublicPath
+      : '/';
```

Comment:
Nits: how about this?

```suggestion
options.contentBasePublicPath = options.contentBasePublicPath || '/';
```

Reply:
Just followed `options.contentBase` initialisation

Reply:
Fixed.
ps: `options.contentBase` is using explcit comparison, because can have number value, and on `0` is resolved as no value.